integrations/zoom/syncs/recording-files.ts

```diff
@@ -0,0 +1,53 @@
+import type { NangoSync, ProxyConfiguration, RecordingFile } from '../../models';
+import type { ZoomRecordingMeeting } from '../types';
+
+export default async function fetchData(nango: NangoSync) {
+    const today = new Date();
+    const monthAgo = new Date(new Date().setMonth(today.getMonth() - 1));
+
+    const config: ProxyConfiguration = {
+        // https://developers.zoom.us/docs/api/meetings/#tag/cloud-recording/GET/users/%7BuserId%7D/recordings
+        endpoint: '/users/me/recordings',
+        params: {
+            from: monthAgo.toISOString().split('T')?.[0] || '',
+            to: today.toISOString().split('T')?.[0] || ''
+        },
+        retries: 10,
+        paginate: {
+            type: 'cursor',
+            cursor_name_in_request: 'next_page_token',
+            cursor_path_in_response: 'next_page_token',
+            response_path: 'meetings',
```

Comment:
You can just include the 
response_path: 'meetings'

since the zoom provider has the other configuration already: https://github.com/NangoHQ/nango/blob/master/packages/shared/providers.yaml#L6821-L6823